src/polygon.jl

```diff
@@ -0,0 +1,29 @@
+# Find the polygon of maximal area, among polygons with nv sides and    
+#   diameter d <= 1
+
+#   This is problem 1 in the COPS (Version 3) collection of 
+#   E. Dolan and J. More'
+#   see "Benchmarking Optimization Software with COPS"
+#   Argonne National Labs Technical Report ANL/MCS-246 (2004)
+
+function polygon_model(args...; n::Int = default_nvar, kwargs...)
```

Comment:
As in the previous function, I would suggest making all the arguments explicit. How about using? 
```julia
function polygon_model(n::Int)
```